packages/scripts/scripts/build.js

```diff
@@ -0,0 +1,46 @@
+const { webpackConfig } = require( '@wordpress/build-config' );
+
+process.env.BABEL_ENV = 'production';
+process.env.NODE_ENV = 'production';
+
+const webpack = require( 'webpack' );
+const formatWebpackMessages = require( 'react-dev-utils/formatWebpackMessages' );
+
+webpackConfig.module.rules.forEach(
```

Comment:
See my PR: https://github.com/WordPress/gutenberg/pull/12837.

I think we should tackle `@wordpress/scripts` separately. It would be awesome to expose Webpack config first.